### PR TITLE
Update add-to-app iOS instructions

### PR DESCRIPTION
**Pull request by p-mazhnik**
## Description
- Remove the deprecated `--universal` and `--xcframework` flags
- Add note about `[build mode]` value
- Remove FlutterPluginRegistrant from Embed Frameworks in image

New image:
embed-xcode

### CODE REVIEW

**Comment by sfshaza2** [comment]
@kenzieschmoll, can you confirm this PR or delegate?

File: src/docs/development/add-to-app/ios/project-setup.md [hunk 1]
```diff
@@ -337,6 +337,10 @@ Host apps using CocoaPods can add Flutter to their Podfile:
 ```ruby
 pod 'Flutter', :podspec => 'some/path/MyApp/Flutter/[build mode]/Flutter.podspec'
 ```
+{{site.alert.note}}
+  You need to hard code `[build mode]` value.
+  For example, you can use `Debug` if you need to use `flutter attach` and `Release` when you're ready to ship.
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/project-setup.md:342]:
```suggestion
  You must hard code  the `[build mode]` value.
  For example, use `Debug` if you need to use
  `flutter attach` and `Release` when you're ready to ship.
```

**Comment by kenzieschmoll** [comment]
> @kenzieschmoll, can you confirm this PR or delegate?

@jonahwilliams do you have familiarity with the content here or know who does?

**Review by sfshaza2 — APPROVED** [review]
LGTM

**Pull request merged**